appveyor.yml

```diff
@@ -32,3 +34,35 @@ deploy:
       skip_symbols: true
       on:
           appveyor_repo_tag: true

```

Comment:
Alternatively, we could combine `deploy` and `after_deploy` into one `deploy_script`. The NuGet API key could move up to `environment`, and pushing to NuGet could be a one-liner inside the `ForEach-Object` below.